templates/page.html

```diff
@@ -276,12 +276,12 @@ <h3>Developers</h3>
         <div class="release-news">
           <img src="{{ '/static/img/release.png'|asseturl }}" />
           <div class="release-news-body">
-            <h3>REANA 0.8.0</h3>
-            <h6>30 Nov 2021</h6>
-            <p>Introduces support for running Snakemake workflows.</p>
-            <p>Improves cluster performance when running multiple parallel workflows.</p>
-            <p>Enables administrators to configure CPU and Disk user quotas.</p>
-            <p class="release-news-link"><a href="https://github.com/reanahub/reana/releases/tag/0.8.0">See full release notes</a></p>
+            <h3>REANA 0.8.1</h3>
+            <h6>15 Feb 2022</h6>
+            <p>Introduces support for job timeout limits.</p>
+            <p>Improves Cluster Health status page on the web interface.</p>
```

Comment:
What about:

- Introduces custom job timeout limits for user workflows.
- Allows searching workspace files by name and visualising HTML reports.
- Improves Cluster Health status web page indicating cluster availability.

Reply:
WRT illustrative picture, the cluster health could do, but I think that the screenshot of the custom job timeouts (as I have added it to the blog post) or of workspace file name search filtering (as you had it in the blog post) might be more illustrative.

(The thinking being that these features will be more frequently used by the researchers than checking out the cluster health page.)

Reply:
Updated.

Replaced release picture with workspace search one.